server/api/products.js

```diff
@@ -0,0 +1,62 @@
+const router = require('express').Router()
+const {Product, Order, Review} = require('../db/models')
+const {Op} = require('sequelize')
+
+//NOTE: ADD BELOW TO API/INDEX.JS
+//router.use('/products', require('./products'))
+
+//NOTE: WILL ADD PAGINATION LATER
+
+//api/products route
+router.get('/', async (req, res, next) => {
+  try {
+    const products = await Product.findAll()
+    res.json(products)
+  } catch (err) {
+    next(err)
+  }
+})
+
+router.post('/', async (req, res, next) => {
+  try {
+    const product = await Product.create({
+      visible: req.body.stock > 0,
```

Comment:
I feel like visibility is seperate from out of stock. In my mind i'm thinking that we can still display out of stock items. Visibility is like its no longer offered.